src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -1194,23 +1192,48 @@ internal StackFrame CreateStackFrame(StackFrameVersionOne v1StackFrame)
             return stackFrame;
         }
 
+        private const string DottedQuadFileVersionPattern = @"[0-9]+(\.[0-9]+){3}";
```

Comment:
>@"[0-9]+(\.[0-9]+){3}" [](start = 60, length = 22)

I'm guessing we don't care, but should this be restricted to 16 bit integers?

Reply:
You're right, that exceeds my care factor :-)

---
In reply to: [314363933](https://github.com/microsoft/sarif-sdk/pull/1647#discussion_r314363933) [](ancestors = 314363933)

Reply:
Come to think of it, this is by design. Although Windows and .NET version numbers are our canonical example of this format, other similarly formatted version numbers might not be so restricted. `dottedQuadFileVersion` by design accepts anything that looks like `n.n.n.n`.

---
In reply to: [314371625](https://github.com/microsoft/sarif-sdk/pull/1647#discussion_r314371625) [](ancestors = 314371625,314363933)